guides/source/active_job_basics.md

```diff
@@ -330,6 +330,31 @@ class GuestsCleanupJob < ActiveJob::Base
 end
 ```
 
+### Deserialization
+
+GlobalID, which Active Job uses, allows serializing full ActiveRecord objects as an argument to `#perform`.
+
+```ruby
+def perform(user)
+  user.send_welcome_email!
+end
+```
+
+if the `User` record is deleted after the job is enqueued but before the perform method is called
+Active Job will raise `ActiveJob::DeserializationError` for a missing record exception
+as part of deserializing the `User` instance.
+
+You can handle this with:
+
+```ruby
+class NotifyUsersJob < ActiveJob::Base
```

Comment:
You can certainly remove this code example as well. Actually `rescue_from` is already explained in the section above.